install_config/web_console_customization.adoc

```diff
@@ -174,6 +174,36 @@ assetConfig:
 ====
 endif::[]
 
+[[enable-tech-preview-feature]]
+=== Enabling Features in Tech Preview
+
+Sometimes features will be available in tech preview, by default these features may be hidden
+from end users.  
+
+The current list of features in tech preview is:
+
+* pipelines - Enabling this will add the Pipelines navigation item underneath the Builds menu
```

Comment:
I'm not sure if this is the right place to list the features currently in tech preview. We do have a place for this in the release notes:

https://docs.openshift.com/enterprise/3.2/release_notes/ose_3_2_release_notes.html#ose-32-technology-preview

If the current features are not accurate this should probably be changed.